docs/docs/sample-scripts/teams/list-all-channels-teams.md

```diff
@@ -0,0 +1,52 @@
+# List all channels in Microsoft Teams team in the tenant
+
+List all channels in Microsoft Teams team in the tenant and exports the results in a CSV.
+
+```powershell tab="PowerShell"
+function Get-Channels(
```

Comment:
The function is defined but never called in the script, is this expected?

Reply:
@garrytrinder yes I had just updated the sample with just the function. Do we need to include how to call the function?

Reply:
Yes, lets be explicit and include an example of how the function can be called